common/cpp/react/renderer/components/rnscreens/RNSScreenComponentDescriptor.h

```diff
@@ -26,13 +38,139 @@ class RNSScreenComponentDescriptor final
             shadowNode.getState());
     auto stateData = state->getData();
 
+#ifdef ANDROID
     if (stateData.frameSize.width != 0 && stateData.frameSize.height != 0) {
+      // When we receive dimensions from JVM side we can remove padding used for
+      // correction, and we can stop applying height correction for the frame.
+      // We want to leave top offset correction though intact.
+      // TODO: In future, when we have dynamic header height we might want to
+      // update Y offset correction here.
+
+#ifdef REACT_NATIVE_DEBUG
```

Comment:
Is it a global flag that is always present?

Reply:
Available since 2021 in RN. It is defined with respect of `NDEBUG` <- this is a standardised macro by the language spec, so we can use that one if you prefer, but I think we should be fine with `REACT_NATIVE_DEBUG` also.

Reply:
Just make sure to use a flag that won't be dropped soon 😄

Reply:
No deprecation mark here